.github/workflows/stale.yml

```diff
@@ -25,7 +25,9 @@ jobs:
       - uses: actions/stale@v8
         with:
           stale-pr-message: 'This PR is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 10 days.'
-          close-pr-message: 'This PR was closed because it has been stalled for 10 days with no activity.'
-          days-before-pr-stale: 45
-          days-before-pr-close: 10
+          close-pr-message: 'This PR was auto-closed because it has been stalled for 10 days with no activity. Please feel free to reopen if it's still valid. Thanks.'
+          stale-issue-message: 'This issue is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 10 days.'
+          close-issue-message: 'This issue was auto-closed because it has been stalled for 10 days with no activity. Please feel free to reopen if it's still valid. Thanks.'
+          days-before-stale: 45
+          days-before-close: 10
```

Comment:
I actually prefer do not close issues

Reply:
`actions/stale` closes issues by default... So let's disable it in this PR?

Reply:
e.g. [this one](https://github.com/oap-project/gluten/issues/3235#event-11090616372)